kolibri/locale/es_ES/LC_MESSAGES/django.po

```diff
@@ -3,7 +3,7 @@ msgstr ""
 "Project-Id-Version: kolibri\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-05-30 19:55-0700\n"
-"PO-Revision-Date: 2018-06-21T02:38:49+0000\n"
```

Comment:
@radinamatic noticing now that we are tracking the Spanish `django.po` file - any reason why? It seems like a mistake, since we normally ignore `.po` files (to avoid inconsistency with CrowdIn)